llvm/test/CodeGen/Mips/fp-maximumnum-minimumnum.ll

```diff
@@ -1,5 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
 ; RUN: llc --mtriple=mipsisa32r6 < %s | FileCheck %s --check-prefix=MIPS32R6
+; RUN: llc --mtriple=mips64 < %s | FileCheck %s --check-prefix=MIPS64R2
```

Comment:
Should precommit the addition of a new run line, I'm not sure if I'm observing the codegen change or not